peachjam/templates/peachjam/judgment_detail.html

```diff
@@ -76,3 +76,11 @@
     {% endif %}
   {% endwith %}
 {% endblock %}
+{% block extra-metadata %}
+  {% if document.headnote_holding %}
+    <dt>
```

Comment:
a `dt` isn't valid outside a `dl`. So you'll need to wrap this in a `dl` and the content of the headnote_holding in a `dd`.

Reply:
Done :heavy_check_mark: